src/components/summaryDetail/SummaryDetail.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import './SummaryDetail.css';
+import { FiChevronUp, FiChevronDown, FiVideo } from 'react-icons/fi';
+
+import Ingredients from './Ingredients';
+import Nutrition from './Nutrition';
+import Instructions from './Instructions';
+import Categories from './Categories';
+
+export default function SummaryDetail({
+  recipeDetail, setRecipeDetail, isDetailShown, toggleIsDetailShown, handleSummaryDetailClose,
```

Comment:
1. setRecipeDetail and toggleIsDetailShown are not being used in the logic, so I guess there is no need for them to pass as a function parameter.

Reply:
Addressed in commit [8eb5dbd](https://github.com/chingu-voyages/v46-tier2-team-20/pull/51/commits/8eb5dbdf846190aaed62d1d8c0df0ec0033b39a5), below